src/module/actor/army/sheet.ts

```diff
@@ -32,6 +38,15 @@ class ArmySheetPF2e extends ActorSheetPF2e<ArmyPF2e> {
         const actor = this.actor;
         const campaignFeatures = actor.itemTypes.campaignFeature;
 
+        const pack = game.packs.get("pf2e.kingmaker-features");
+        const compendiumFeatures = ((await pack?.getDocuments({ type: "campaignFeature" })) ?? []).filter(
```

Comment:
Can this be cached?

Reply:
I could, but I was under the assumption the compendiums were already caching that.

Reply:
Ah wait but yeah, I'm cloning. Nvm I will cache